lib/src/common/locator/service_locator.dart

```diff
@@ -10,4 +12,5 @@ void registerServices() {
 
 void _registerExampleService() {
   locator.registerLazySingleton<ExampleService>(() => ExampleServiceImpl());
+  locator.registerLazySingleton<AuthenticationService>(() => AuthenticationServiceImpl());
```

Comment:
better create another function named `_registerAuthenticationService` and call it inside `registerServices` function.